jina/peapods/networking.py

```diff
@@ -388,11 +396,13 @@ async def task_wrapper(new_messages, stub):
                 except AioRpcError as e:
                     if e.code() != grpc.StatusCode.UNAVAILABLE or i == 2:
                         raise
+                    elif e.code() == grpc.StatusCode.UNAVAILABLE and i == 2:
```

Comment:
this code is not reachable right? `i==2` will always activate the first `if`

Reply:
True, will fix it. I wanted to print the log message here